Emby.Dlna/Didl/DidlBuilder.cs

```diff
@@ -232,12 +232,15 @@ private void AddVideoResource(DlnaOptions options, XmlWriter writer, BaseItem vi
                 AddVideoResource(writer, video, deviceId, filter, contentFeature, streamInfo);
             }
 
-            var subtitleProfiles = streamInfo.GetSubtitleProfiles(_mediaEncoder, false, _serverAddress, _accessToken)
-                .Where(subtitle => subtitle.DeliveryMethod == SubtitleDeliveryMethod.External)
-                .ToList();
+            var subtitleProfiles = streamInfo.GetSubtitleProfiles(_mediaEncoder, false, _serverAddress, _accessToken);
```

Comment:
Is LINQ Where actually slower than the filter in the foreach? I thought they were lazily evaluated.

Reply:
Linq is always (slightly) slower due to their implementation with generics.

Reply:
I feel like we're sacrificing readability for micro-optimisations here. Linq isn't always slower either as it is lazily evaluated.

Reply:
I agree. The most subtitles I've seen in a single file is ~40. I don't think readability is hurt all that much by this particular change, but generally I do prefer smaller LINQ expressions over the more terse foreach loops.

Reply:
I find it more readable, but If you guys disagree I'l change it.

Reply:
The big change here was the removal of the `ToList`, not the move of the `Where`